test/util/Util.ts

```diff
@@ -43,6 +49,15 @@ export function getPort(name: typeof portNames[number]): number {
   return 6000 + idx;
 }
 
+export function getSocket(name: typeof socketNames[number]): string {
+  const idx = socketNames.indexOf(name);
+  // Just in case something doesn't listen to the typings
+  if (idx < 0) {
+    throw new Error(`Unknown socket name ${name}`);
+  }
+  return resolve(`./test/tmp/css${idx}.sock`);
```

Comment:
There is a `getTestFolder` utility function that generates a path to a folder where all test data is placed. Should use that one for consistency. Also want to remove that folder afterwards with an `afterAll` call in the relevant test using the `removeFolder` utility function.